### ISSUE TITLE
Scratch not working; red crosses appearing all over text

### ISSUE THREAD

**Issue by Eribetra** [raspberrypi/scratch#235]
Scratch is displaying red crosses all over what should be text!

The error is: primSetFont:size:bold:italic:antialias: failed

Maybe it has to do with fonts?

**Comment by timrowledge** [comment]
Yes, it has to do with fonts. At a guess you don't have the font files that Cairo/Pango requires and that is a bit of a puzzle because they should all be there by default. Have you changed the language setting at all? I could imagine that some of the language choices would have problems with missing fonts - or characters within the fonts I suppose - and we never could thoroughly test every possible language.

If you could post more info about your set up, and perhaps the 'SqueakDebug.log' file (or something named like that) it might help.

**Comment by Laphatize** [comment]
Usually, this happens when the language isn't supported by the font.
There really isn't any fix for it...
